server/views/components/numbered-list/index.njk

```diff
@@ -17,10 +17,18 @@
       {% else %}
         <span class="numbered-list__link-surrogate">
       {% endif %}
-          <h3 class="numbered-list__title">{{ item.title }}</h3>
+          <h3 class="numbered-list__title">
+            {%- if current === item.url -%}
+              Currently viewing<span class="numbered-list__visually-hidden-title"> {{ item.title }}</span>
```

Comment:
We don't – for reasons of things being visually shown/hidden at different breakpoints. Perhaps we should, though.